map/map_height_fitter/src/map_height_fitter.cpp

```diff
@@ -186,7 +192,14 @@ std::optional<Point> MapHeightFitter::Impl::fit(const Point & position, const st
     tf2::Transform transform{tf2::Quaternion{}, tf2::Vector3{}};
     tf2::fromMsg(stamped.transform, transform);
     point = transform * point;
-    point.setZ(get_ground_height(point));
+
+    //
+    auto point_optional = get_ground_height(point);
```

Comment:
This value can be `const` and is not a `point`.

Reply:
I made it const and modified the name in  [92be65d](https://github.com/autowarefoundation/autoware.universe/pull/4199/commits/92be65d6469a65c9de6546a861b84e4633e89845)